mobx/lib/mobx.dart

```diff
@@ -61,3 +61,4 @@ export 'package:mobx/src/core.dart'
         NotificationHandlers,
         ReactionImpl,
         DerivationState;
+const version='0.3.7';
```

Comment:
Should probably do this as the top line after the `library` statement ? Easier to spot then. Same for all other packages

Reply:
It was easier to append to the file, because of the bash script 😄 . I plea guilty for being lazy 😉

Reply:
you got something working, which is a great step forward! The suggestion is just a tiny improvement to what you have done :-)

Reply:
Hm, I cannot actually put it anywhere else but the bottom of the file because of `Directives must appear before any declaration`.